src/hooks/useDraftUpdater.ts

```diff
@@ -41,17 +41,28 @@ function useDraftUpdater() {
                 return Promise.resolve();
             }
 
+            const updatedSpec = updateSpec(cloneDeep(draftSpec.spec));
+
+            if (updatedSpec === undefined) {
+                // Means we are trying to update something on the spec that isn't there yet so just store off
+                //  into the store and keep carrying on
+                logRocketEvent('PropertyUpdater', {
+                    updatedSpecMissing: true,
+                });
+
+                // If we do not have these yet do not fail - just keep going and hopefully the calling code
+                //  will save the value into a store
+                return Promise.resolve();
+            }
```

Comment:
New functionality that should have always been there. We need a way for the specific calls to tell the update function to skip the update without error. This is like how `useTimeTravel` checks if the binding is in a spec or we just need to store things in a store.